charts/mirrorbits-parent/templates/_helpers.tpl

```diff
@@ -40,3 +40,70 @@ Must be 63 chars. maximum.
 {{- define "mirrorbits-parent.pv-secretname" -}}
 {{- printf "%s-%s" (include "mirrorbits-parent.name" . | trunc 43 ) "persistentvolume-secret" -}}
 {{- end -}}
+
+{{/*
+Define the azure file persistent volume claim name (if enabled).
+Must be 63 chars. maximum.
+*/}}
+{{- define "mirrorbits-parent.pvc-name" -}}
+{{- printf "%s-%s" .Release.Name "mirrorbits-parent-data" -}}
+{{- end -}}
+
+{{/*
+This template method checks the current ingress path and fail with a user facing message if needed or return empty.
```

Comment:
```suggestion
This template method checks the current ingress path and fails with a user facing message if needed or returns empty.
```
I'm not sure if a comma is needed or not after the `or` 🤔

Reply:
> I'm not sure if a comma is needed or not after the or 🤔 

I believe it is, yes